neon_local/src/main.rs

```diff
@@ -540,6 +540,33 @@ fn handle_tenant(tenant_match: &ArgMatches, env: &mut local_env::LocalEnv) -> an
                 "tenant {} successfully created on the pageserver",
                 new_tenant_id
             );
+
+            // Create an initial timeline for the new tenant
+            let new_timeline_id = parse_timeline_id(create_match)?;
+            let timeline = pageserver
+                .timeline_create(new_tenant_id, new_timeline_id, None, None)?
+                .ok_or_else(|| {
```

Comment:
NIT:

1. You can put identifiers (not expressions though) into `{}` brackets now:
` "Failed to create initial timeline for tenant {new_tenant_id}"` note that expressions are still not supported and something like this won't work: ` "Failed to create initial timeline for tenant {new_tenant_id.to_string()}"`

2. You can apply `anyhow::Context` trait methods onto `Option` directly:
`.timeline_create(new_tenant_id, new_timeline_id, None, None)?.with_context(|| format!("Failed to create initial timeline for tenant {new_tenant_id}"))?`

Reply:
> NIT:
> 
> 1. You can put identifiers (not expressions though) into `{}` brackets now:
>    ` "Failed to create initial timeline for tenant {new_tenant_id}"` note that expressions are still not supported and something like this won't work: ` "Failed to create initial timeline for tenant {new_tenant_id.to_string()}"`
> 2. You can apply `anyhow::Context` trait methods onto `Option` directly:
>    `.timeline_create(new_tenant_id, new_timeline_id, None, None)?.with_context(|| format!("Failed to create initial timeline for tenant {new_tenant_id}"))?`

TIL about using `anyhow::Context` with `Option` :eyes:. Thanks